lib/scope.js

```diff
@@ -144,7 +145,7 @@ ScopeDefinition.prototype.related = function(receiver, scopeParams, condOrRefres
           queryRelated.where[IdKey] = collectTargetIds(data, IdKey);
         }
 
-        relatedModel.find(queryRelated, cb);
+        relatedModel.find(queryRelated, options, cb);
```

Comment:
HI @jannyHou,
why we should remove the options in line 109? *.
Let's assume that I want to use the options in every **.observe('access', ..)  (every model)
This would no longer be possible.